smoketest/scripts/cli/test_service_dhcp-server.py

```diff
@@ -1366,7 +1366,7 @@ def test_dhcp_hostsd_lease_sync(self):
         def internal_cleanup():
             for seq in client_range:
                 ip_addr = inc_ip(subnet, seq)
-                kea_delete_lease(4, ip_addr)
+                kea_delete_lease(4, '', ip_addr)
```

Comment:
```suggestion
                kea_delete_lease(4, None, ip_addr)
```

We should rather pass `None` then an empty string.